src/components/styled/PinkTypography.ts

```diff
@@ -0,0 +1,10 @@
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+const StyledTypography = styled(Typography)(({ theme }) => ({
+  color: theme.palette.primary.main,
+  fontWeight: 900,
+  padding: "34px 0 40px 0",
+  width: "100%",
+}));
+export default StyledTypography;
```

Comment:
I'm not sure this necessarilly needs to be it's own component. I think we may end up having too many components if we create one for each text variation in the app.

It would be better to update the theme with the styling for an `h1` tag, reference that, and then use the standard MUI Typography component. E.g.

```
// theme.ts

export default createTheme({
  ...
  typography: {
  ...
  h1: {
    fontFamily: "Raleway",
    fontSize: "34px",
    fontWeight: 900,
  })
  ...
})

```

```
...
// Songs.tsx

  ...
  <Box pt="34px" pb="40px">
    <Typography variant="h3" color="primary" align="center" /> 
  </Box>
  ...
```

Reply:
We might not want to apply these styles to every Typography h1. Would it be better to just make the styles inline or what do you think. For now Ill just commit them inline but let me know if you want me to change that.

Reply:
I'm fine with making them inline. It looks like in this case, we're actualy just adding some custom styles to the existing `h5` styles, which I think is completely valid.